packages/fxa-settings/src/pages/Signin/SigninUnblock/index.tsx

```diff
@@ -133,7 +133,15 @@ const SigninUnblock = ({
         queryParams: location.search,
       };
 
-      await handleNavigation(navigationOptions, true);
+      const { error: navError } = await handleNavigation(
+        navigationOptions,
+        true
+      );
+      if (navError) {
+        setBannerErrorMessage(
+          getLocalizedErrorMessage(ftlMsgResolver, navError)
+        );
+      }
```

Comment:
If `data` exists from `signinWithUnblockCode`, then `error` must be undefined, so that should never happen here.

I mentioned this above, but I'm not sure how I'm feeling about this pattern now. In some ways, returning an undefined error is nice. In other ways, `try`/`catch` seems less confusing?